src/components/footer/footer.js

```diff
@@ -1,41 +1,41 @@
-export default class FooterComponent extends HTMLElement {
-  // run some code to set HTML when the component is ready
-  connectedCallback() {
-    this.innerHTML = this.getTemplate();
-  }
+// create templates that interpolate variables and HTML!
+const template = document.createElement('template');
+const year = new Date().getFullYear();
+template.innerHTML =  `
+<style>
+.footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background-color: #192a27;
+  min-height: 30px;
+  padding-top: 10px;
+}
+.footer a {
+  color: #efefef;
+  text-decoration: none;
+}
+.footer h4 {
+  width: 90%;
+  margin: 0 auto;
+  padding: 0;
+  text-align: center;
+}
+</style>
+
+<footer class="footer">
+  <h4>
+    <a href="https://www.greenwoodjs.io/">My Blog &copy;${year} &#9672 Built with GreenwoodJS</a>
+  </h4>
+</footer>
+`;
 
-  // create templates that interpolate variables and HTML!
-  getTemplate() {
-    const year = new Date().getFullYear();
 
-    return `
-      <style>
-      .footer {
-        position: fixed;
-        bottom: 0;
-        width: 100%;
-        background-color: #192a27;
-        min-height: 30px;
-        padding-top: 10px;
-      }
-      .footer a {
-        color: #efefef;
-        text-decoration: none;
-      }
-      .footer h4 {
-        width: 90%;
-        margin: 0 auto;
-        padding: 0;
-        text-align: center;
-      }
-      </style>
 
-      <footer class="footer">
-        <h4>
-          <a href="https://www.greenwoodjs.io/">My Blog &copy;${year} &#9672 Built with GreenwoodJS</a>
-        </h4>
-      </footer>
-    `;
+export default class FooterComponent extends HTMLElement {
+  // run some code to set HTML when the component is ready
+  connectedCallback() {
+    this.innerHTML = template.content.cloneNode(true);
```

Comment:
Yeah, this doesn't seem to work like in my examples for DSD from the issue and is breaking the build for me locally too. 

But as a quick way forward, this does work at least
```js
export default class HeaderComponent extends HTMLElement {
  connectedCallback() {
    this.innerHTML = template.innerHTML;
  }
}
```

Getting closer to DSD conventions this works...
```js
export default class HeaderComponent extends HTMLElement {
  connectedCallback() {
    this.appendChild(template.content.cloneNode(true));
  }
}
```

However, this will render everything out into a DSD (e.g. `<template shadowrootmode="open">...</template>` (naturally) which is not really what we want here since we are explicitly looking to render out only to Light DOM for these components.

----

So yeah, I think this is the form we want to go with I think
```js
export default class HeaderComponent extends HTMLElement {
  connectedCallback() {
    this.innerHTML = template.innerHTML;
  }
}
```